### ISSUE TITLE
sendLogToLastFailedTest and sendFileToLastFailedTest send data to different test

### ISSUE THREAD

**Issue by igerasimov23** [BorisOsipov/wdio-reportportal-reporter#18]
## The problem

When running tests with retry option and tests are failing sendLogToLastFailedTest and sendFileToLastFailedTest send data to different test. Looks like it sends data to [test - 1] or the current test

## Environment

* WebdriverIO version: 4.11
* wdio-reportportal-version version: 0.0.13
* Node.js version: 8.10
* WDIO Testrunner and version(mocha\cucumber\jasmine):
* Additional wdio packages used (if applicable):

## Code To Reproduce Issue [ Good To Have ]

```
describe('test footer', function () {
    this.retries(2);

    beforeEach(() => {
        browser.url('some url');
    }, 2);

    it('Verify footer is displayed -feature footer', function () {
        assert.isTrue(false, 'test 1 failed')
    });

    it('Verify footer buttons navigation -env prod -feature footer', function () {
      browser.url('some url');
      assert.isTrue(false, 'test 2 failed')
    })

});
```
And basic after test hook: 
```
afterTest: function (test) {
    //logging Pass or Fail for test
    (test.passed) ? reporter.sendLogToLastFailedTest('debug', `******* TEST '${test.title}' PASSED ******* `)
      : reporter.sendLogToLastFailedTest('debug', `******* TEST '${test.title}' FAILED ******* `);
    if (test.passed === false) {
      const screenshot =  browser.saveScreenshot();
      reporter.sendFileToLastFailedTest('info', 'failed.png', screenshot);
    }
  },
```

Adding screenshots to better understand problem:
1st test:
<img width="1370" alt="screen shot 2018-10-09 at 1 56 38 pm" src="https://user-images.githubusercontent.com/15241367/46689226-5df6f500-cbcd-11e8-92f6-7e88aced63f7.png">

2nd test:
<img width="1362" alt="screen shot 2018-10-09 at 1 57 05 pm" src="https://user-images.githubusercontent.com/15241367/46689243-68b18a00-cbcd-11e8-8bd2-ae3af3661531.png">

Second test displays correct sendLogToLastFailedTest() data and screenshots, but first test contains sendLogToLastFailedTest() and screenshots from first tests as well as from second test.

Let me know if more input needed from my side

**Comment by BorisOsipov** [comment]
Meh. I will see what we can do with it. Thanks for detailed issue report!

**Comment by BorisOsipov** [comment]
Fixed in https://github.com/BorisOsipov/wdio-reportportal-reporter/pull/15

**Comment by igerasimov23** [comment]
I will finish testing by end of the week and let you know  as can not do earlier :(

**Comment by BorisOsipov** [comment]
>Sorry can not do earlier :(

No problem. I tested a bit in m samples and it works good. I released it with 0.0.14
